src/components/Box/Box.types.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
+import { DefaultComponentProps } from '../../types';
+
+export interface BoxProps extends DefaultComponentProps {
+	condensed?: boolean;
+	backgroundColor?: 'gray' | 'soft-white' | 'white' | 'dark';
+	children: ReactNode;
```

Comment:
children hoef je normaal niet zelf te typen dit wordt opgevangen in React zijn `FC` type